app/scripts/controllers/eventDetails.js

```diff
@@ -227,7 +227,7 @@ angular.module('confRegistrationWebApp')
 
       //Minimum Deposit
       angular.forEach($scope.conference.registrantTypes, function(t) {
-        if ($scope.conference.requireLogin && $scope.anyPaymentMethodAccepted(t) && String(t.minimumDeposit).length > 0 && !_.isNull(t.minimumDeposit)) {
+        if (($scope.conference.relayLogin || $scope.conference.facebookLogin) && $scope.anyPaymentMethodAccepted(t) && String(t.minimumDeposit).length > 0 && !_.isNull(t.minimumDeposit)) {
```

Comment:
It may be a good idea to move this check `$scope.conference.relayLogin || $scope.conference.facebookLogin` into an attribute on conference.